### PR TITLE
TPROD: 2-Fix entity names using rector

### PR DESCRIPTION
**Pull request by mabumusa1**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. **Code review is required, once CI goes green this PR is good to for code review**

### CODE REVIEW

**Comment by mabumusa1** [comment]
@escopecz After this PR is merged, we will have the following rules to run,  I will create a small PR to merge rector rules into sets

```
To be applied
AddEntityIdByConditionRector
ClassAnnotationToNamedArgumentConstructorRector
ManagerRegistryGetManagerToEntityManagerRector
MoveRepositoryFromParentToConstructorRector
ReplaceLifecycleEventArgsByDedicatedEventArgsRector
ReplaceParentRepositoryCallsByRepositoryPropertyRector
ServiceEntityRepositoryParentCallToDIRector

Not found in the current version of rector
ChangeCompositeExpressionAddMultipleWithWithRector
RemoveEmptyTableAttributeRector
```

File: app/bundles/CampaignBundle/Entity/LeadEventLog.php [hunk 1]
```diff
@@ -16,12 +16,12 @@ class LeadEventLog implements ChannelInterface
...
 
     /**
-     * @var Event|null
+     * @var Event
      */
     private $event;
 
     /**
-     * @var LeadEntity|null
+     * @var LeadEntity
```
Comment (escopecz) [inline app/bundles/CampaignBundle/Entity/LeadEventLog.php]: This should be nullable

File: app/bundles/CampaignBundle/Entity/LeadEventLog.php [hunk 2]
```diff
@@ -16,12 +16,12 @@ class LeadEventLog implements ChannelInterface
     private $id;
 
     /**
-     * @var Event|null
+     * @var Event
```
Comment (escopecz) [inline app/bundles/CampaignBundle/Entity/LeadEventLog.php]: This should be nullable

File: app/bundles/CampaignBundle/Entity/LeadEventLog.php (hunk 1 above)
Comment (escopecz) [inline app/bundles/CampaignBundle/Entity/LeadEventLog.php]: This should be nullable

File: app/bundles/CampaignBundle/Entity/LeadEventLog.php [hunk 3]
```diff
@@ -61,7 +61,7 @@ class LeadEventLog implements ChannelInterface
     private $metadata = [];
 
     /**
-     * @var bool
+     * @var bool|null
```
Comment (escopecz) [inline app/bundles/CampaignBundle/Entity/LeadEventLog.php]: This should not be nullabe. It has a default value of `false`

File: app/bundles/ChannelBundle/Entity/Message.php [hunk 4]
```diff
@@ -19,7 +19,7 @@ class Message extends FormEntity
     private $id;
 
     /**
-     * @var ?string
+     * @var string
```
Comment (escopecz) [inline app/bundles/ChannelBundle/Entity/Message.php]: This should be nullable

File: app/bundles/CoreBundle/Entity/TranslationEntityInterface.php [hunk 5]
```diff
@@ -12,7 +12,7 @@ interface TranslationEntityInterface
     /**
      * Get translation parent.
      *
-     * @return TranslationEntityInterface
+     * @return mixed
```
Comment (escopecz) [inline app/bundles/CoreBundle/Entity/TranslationEntityInterface.php]: This won't work?
```suggestion
     * @return TranslationEntityInterface|null
```

File: app/bundles/CoreBundle/Entity/TranslationEntityTrait.php [hunk 6]
```diff
@@ -16,12 +16,12 @@ trait TranslationEntityTrait
     public $languageSlug;
 
     /**
-     * @var ArrayCollection
+     * @var mixed
```
Comment (escopecz) [inline app/bundles/CoreBundle/Entity/TranslationEntityTrait.php]: This won't work? Similar to other changes in this file
```suggestion
     * @var ArrayCollection<TranslationEntityInterface>
```

File: app/bundles/CoreBundle/Entity/VariantEntityInterface.php [hunk 7]
```diff
@@ -12,7 +12,7 @@ interface VariantEntityInterface
     /**
      * Get translation parent.
      *
-     * @return VariantEntityInterface
+     * @return mixed
```
Comment (escopecz) [inline app/bundles/CoreBundle/Entity/VariantEntityInterface.php]:
```suggestion
     * @return VariantEntityInterface|null
```

File: app/bundles/CoreBundle/Entity/VariantEntityTrait.php [hunk 8]
```diff
@@ -4,17 +4,18 @@
...
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping\Entity;
 use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
 
 trait VariantEntityTrait
 {
     /**
-     * @var ArrayCollection
+     * @var mixed
```
Comment (escopecz) [inline app/bundles/CoreBundle/Entity/VariantEntityTrait.php]: This won't work? Similar to other changes in this file
```suggestion
     * @var ArrayCollection< VariantEntityInterface >
```

File: app/bundles/EmailBundle/Entity/Stat.php [hunk 9]
```diff
@@ -31,7 +31,7 @@ class Stat
     private $lead;
 
     /**
-     * @var string|null
+     * @var string
```
Comment (escopecz) [inline app/bundles/EmailBundle/Entity/Stat.php]: Should be nullable

File: app/bundles/LeadBundle/Entity/Company.php [hunk 10]
```diff
@@ -23,12 +23,12 @@ class Company extends FormEntity implements CustomFieldEntityInterface, Identifi
     private $id;
 
     /**
-     * @var int
+     * @var int|null
```
Comment (escopecz) [inline app/bundles/LeadBundle/Entity/Company.php]: This should not be nullable as it has default value of `0`

File: app/bundles/LeadBundle/Entity/CompanyLead.php [hunk 11]
```diff
@@ -23,7 +23,7 @@ class CompanyLead
     private $dateAdded;
 
     /**
-     * @var bool
+     * @var bool|null
```
Comment (escopecz) [inline app/bundles/LeadBundle/Entity/CompanyLead.php]: This should not be nullable as it has a default value of `false`

**Review by escopecz — CHANGES_REQUESTED** [review]
@mabumusa1 I did another part of review. I still cannot get through it all. I'll continue another day to get to other tasks. If you'll see similar issues in the diff, please change them.

**Review by escopecz — APPROVED** [review]
I see no problem with these changes. Thanks for replacing those strings! 👍

**Pull request merged**